### ISSUE TITLE
Moving wand crashing the botania manapool?

### ISSUE THREAD

**Issue by Sasikaa** [romelo333/notenoughwands#9]
When i click the moving wand the mana pool (full of mana!!) crashing my game!
This is my crash log:
http://pastebin.com/DXjUJzSc

**Comment by McJty** [comment]
Yes, some tile entities will not like being moved. This should probably be added to the blacklist

**Comment by Sasikaa** [comment]
please give me a text what put the config blacklist the manapool...
Please!
